src/main/java/bg/libapp/libraryapp/exceptions/OldPublisherEqualsNewPublisherException.java

```diff
@@ -0,0 +1,7 @@
+package bg.libapp.libraryapp.exceptions;
+
+public class OldPublisherEqualsNewPublisherException extends RuntimeException {
```

Comment:
When old and new value match when trying to update, you should simply not update(create audit record) for it instead of throwing an exception